analyses/mwi/dryspells_trigger/exploration/2021_2022_postseason_overview.Rmd

```diff
@@ -0,0 +1,570 @@
+
+
+```{r rainy-days-plots}
+
+rainy_ratio_labels <- c( "0-20", "21-40", "41-60", "61-80", "81-100")
+  
+rain_seas_tot_min <- plyr::round_any(minmax(static_r[["rain_seas_tot"]])[1], 100, f=floor)
+rain_seas_tot_max <- plyr::round_any(minmax(static_r[["rain_seas_tot"]])[2], 100, f=ceiling)
+rain_seas_tot_breaks <- seq(from = plyr::round_any(rain_seas_tot_min, 100, f=floor), 
+                            to = plyr::round_any(rain_seas_tot_max, 100, f=ceiling), # ceiling ensures max value included
+                            by = 100)
+
+par(mfrow = c(1,2),
+    mar = c(1, 1, 3, 1)) # bottom, left, top, right.
+
+plot(static_r, 
+     "rainy_ratio", 
+     breaks = seq(from = 0, to = 100, by = 20), 
+     main = "Rainy Days \n During Rainy Season",
+     axes = F,
+     plg = list(title = "Rainy Days (%)",
+                legend = rainy_ratio_labels),
+     col = RColorBrewer::brewer.pal(n = 9, name = "YlGnBu"))
+plot(southern_shp, add = T)
+
+plot(static_r, 
+     "rain_seas_tot", 
+     breaks = rain_seas_tot_breaks, 
+     main = "Total Rainfall \n During Rainy Season",
+     axes = F,
+     plg = list(title = "Rainfall (mm)"),
+     col = RColorBrewer::brewer.pal(n = 9, name = "Blues"))
+plot(southern_shp, add = T)
+```
+
+### Monthly Patterns
+
+Returning to dry days (ie, with less than 3mm of rain), patterns emerge when their occurrence is examined per calendar month as in the maps below. (Note that the maps include days that preceded or followed season onset or cessation as defined above in order to represent the full calendar month).
+
+The months of January and February saw the fewest dry days, although there were numerous dry days in both months (January: `r minmax(dry_days_per_month_r[['jan']])[1]` - `r minmax(dry_days_per_month_r[['jan']])[2]`. February: `r minmax(dry_days_per_month_r[['feb']])[1]` - `r minmax(dry_days_per_month_r[['feb']])[2]` depending on the area). 
+
+Outside of the observational trigger monitoring period, November, December, March and April were similar in the average number of dry days observed across the Southern Region 
+(`r round(as.numeric(global(dry_days_per_month_r[['nov']], mean, na.rm=T)), 1)`, 
+`r round(as.numeric(global(dry_days_per_month_r[['dec']], mean, na.rm=T)), 1)`, 
+`r round(as.numeric(global(dry_days_per_month_r[['mar']], mean, na.rm=T)), 1)`, 
+`r round(as.numeric(global(dry_days_per_month_r[['apr']], mean, na.rm=T)), 1)`, respectively.)
+
+
+```{r dry-days-per-month-plot}
+
+dry_days_per_month_breaks <- seq(from = 0, to = 32, by = 5)
+
+monthly_patterns_col <- rev(RColorBrewer::brewer.pal(n = 6, name = "YlGnBu"))
+
+par(mfrow = c(1,6),
+    mar = c(0, 0, 1.5, 0)) # bottom, left, top, right.
+
+plot(dry_days_per_month_r, 
+     "nov", 
```

Comment:
This is great, and addresses the comments I put above! I guess the question becomes, is there significant impact of the slightly fewer dry days seen in the south of the southern region in November December that results in an earlier rainy season vs. the more concentrated rainfall in the northern area?